client/scripts/views/pages/landing/community_cards.ts

```diff
@@ -79,55 +87,57 @@ const CommunityCard : m.Component<{ community: CommunityInfo }> = {
     const updatedThreads = unseenPosts[community.id]?.activePosts || 0;
     const monthlyThreadCount = app.recentActivity.getCommunityThreadCount(community.id);
 
-    return m(Card, {
-      elevation: 1,
-      interactive: true,
-      class: 'home-card',
-      onclick: (e) => {
-        e.preventDefault();
+    const redirectFunction = (e) => {
+      e.preventDefault();
         localStorage['home-scrollY'] = window.scrollY;
         m.route.set(`/${community.id}`);
-      },
-    }, [
-      m('.card-left', [
-        m(CommunityIcon, { community }),
+    }
+
+    // Potentially Temporary (could be built into create community flow)
+    let pretty_description = '';
```

Comment:
we can leave a comment-note here for future implementation of pretty_description, but until then, we don't need to ship unnecessary code

Reply:
I think we should leave this for the time being, because it adds a period to the end of the description if none exists. I just meant we could be adding this period enforcement to the form where they initially set the description.